packages/expo-modules-autolinking/src/platforms/android.ts

```diff
@@ -23,18 +23,13 @@ export async function generatePackageListAsync(
   await fs.promises.writeFile(targetPath, generatedFileContent, 'utf8');
 }
 
-async function findGradleFilesAsync(revision: PackageRevision): Promise<string[]> {
-  const configGradlePaths = revision.config?.androidGradlePaths();
-  if (configGradlePaths && configGradlePaths.length) {
-    return configGradlePaths;
-  }
-
-  const buildGradleFiles = await glob('*/build.gradle{,.kts}', {
-    cwd: revision.path,
-    ignore: ['**/node_modules/**'],
-  });
```

Comment:
leaving a behavior changing note here which is not a big problem though.

original we search build.gradle from `node_modules/{pkg}/*/build.gradle` and now we only search it from `node_modules/{pkg}/android/build.gradle` (having default `path=android`)

if an expo module has build.gradle not in a default android directory, they should explicit add `path` in expo-module.config.json

Reply:
Yes, that was intentional. I don't want to guess which library we should include.